### PR TITLE
[CCP]Add schema for response CRD

### PR DESCRIPTION
**Pull request by 3quanfeng**
## Checklist
- [x] The title of the PR is clear and informative
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] Issues addressed by the PR are mentioned in the description followed by `Fixes`.

## Description
Schema and example of AppGw response CRD

### CODE REVIEW

File: crds/AzureApplicationGatewayResponse_v1beta1.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+apiVersion: apiextensions.k8s.io/v1beta1 
+kind: CustomResourceDefinition 
+metadata: 
+  name: azureapplicationgatewayresponses.appgw.ingress.azure.io
```
Comment (akshaysngupta) [inline crds/AzureApplicationGatewayResponse_v1beta1.yaml:4]: How about `azureapplicationgatewayinstancestatus`?

Comment (3quanfeng) [reply crds/AzureApplicationGatewayResponse_v1beta1.yaml]: Updated to azureapplicationgatewayinstanceupdatestatus

File: crds/examples/AzureApplicationGatewayInstanceUpdateStatus.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+apiVersion: appgw.ingress.azure.io/v1beta1
+kind: AzureApplicationGatewayInstanceUpdateStatus
+metadata:
+  name: subscription-resourcegroup-gatewayname-instanceid
+spec:
+  instanceID: "1"
+  desiredBackendPoolResourceVersion: "17358752"
+  currentAppliedBackendPoolResourceVersion: "1700000"
+  lastCheckInTime:  "Wed, Jun 24, 2020  1:45:43 PM"
+  errorMessage: "RELOAD_NGINX_CONF_ERROR"
```
Comment (akshaysngupta) [inline crds/examples/AzureApplicationGatewayInstanceUpdateStatus.yaml:10]: can you can this to "UNABLE_TO_CONNECT_TO_API_SERVER"

Comment (3quanfeng) [reply crds/examples/AzureApplicationGatewayInstanceUpdateStatus.yaml:10]: This is just an example

**Pull request merged**